neo/Prompt/Commands/tests/test_wallet_commands.py

```diff
@@ -34,8 +36,84 @@ def GetWallet1(cls, recreate=False):
                                            to_aes_key(UserWalletTestCase.wallet_1_pass()))
         return cls._wallet1
 
+    @classmethod
+    def OpenWallet(cls):
+        PromptData.Wallet = cls.GetWallet1(recreate=True)
+
+    @classmethod
+    def tearDown(cls):
+        PromptData.Wallet = None
+
     # Beginning with refactored tests
 
+    def test_wallet(self):
+        with patch('neo.Prompt.PromptData.PromptData.Prompt'):
```

Comment:
what are we trying to patch? I don't see it used.

Reply:
Also, one way to make this test less confusing would be `return Prompt data.Wallet` which you could after as true. You could also verify `stdout` if you wanted but not necessary.

Reply:
I also think this test is failing on no wallet